performance_test_fixture-extras.cmake

```diff
@@ -21,9 +21,11 @@ macro(_performance_test_fixture_find_memory_tools)
       osrf_testing_tools_cpp::memory_tools LIBRARY_PRELOAD_ENVIRONMENT_IS_AVAILABLE)
 
     if(NOT _PERFORMANCE_TEST_FIXTURE_MEMORY_TOOLS_AVAILABLE)
-      message(WARNING
-        "'osrf_testing_tools_cpp' memory tools are not available, C++ tests "
-        "using 'performance_test_fixture' can not be run and will be skipped.")
+      if(AMENT_RUN_PERFORMANCE_TESTS)
+        message(WARNING
+          "'osrf_testing_tools_cpp' memory tools are not available, C++ tests "
+          "using 'performance_test_fixture' can not be run and will be skipped.")
+      endif()
     else()
```

Comment:
Should this be an `elseif (AMENT_RUN_PERFORMANCE_TESTS)`? Or is this `get_target_property` needed regardless of `AMENT_RUN_PERFORMANCE_TESTS`?

Reply:
It's definitely not needed, but I think it would be OK to still add the extra environment variable even if we're going to skip the test. That way, the only difference in the command that CTest invokes is the presence of the `--skip-test` argument, and all of the other arguments (the environment vars, executable name, etc) are identical.

tl;dr - it doesn't hurt to leave it.